doc/book/middleware.md

```diff
@@ -53,10 +53,10 @@ $route = Literal::factory([
 
 Middleware may be provided as PHP callables, or as service names.
 
-> ### No action required
->
-> Unlike action controllers, middleware typically is single purpose, and, as
-> such, does require a default `action` parameter.
+### No action required
+
+Unlike action controllers, middleware typically is single purpose, and, as
+such, does not require a default `action` parameter.
```

Comment:
Thanks for fixing the typo. However, this *is* supposed to be done as a note, and we use the blockquote markup for those.

I'll re-instate the blockquotes when merging. Thanks!